project/plugins.sbt

```diff
@@ -8,3 +8,5 @@ addSbtPlugin("org.xerial.sbt" % "sbt-pack" % "0.9.1")
 // TODO: This plugin is part of SBT 1.x. Remove it after upgrading.
 // https://github.com/sbt/sbt-doge/issues/18
 addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC13")
```

Comment:
what's this

Reply:
Hm.  That is me accidentally committing the plugin I use to take resolving time way down.  I will save it for another PR

Reply:
good tease